torchx/schedulers/ray/ray_driver.py

```diff
@@ -127,17 +137,33 @@ def create_command_actors(
 
 
 def main() -> None:  # pragma: no cover
+
     actors: List[RayActor] = load_actor_json("actors.json")
+    if re.match("\\d+:\\d+", actors[0].nnodes_rep):
```

Comment:
outside of the component file we should avoid regular expressions (use a typed min_replicas instead)

Reply:
Done